src/software/simulated_tests/example_play_test.cpp

```diff
@@ -38,7 +39,7 @@ TEST_F(ExamplePlayTest, test_example_play)
                             return false;
                         }
                     }
-                    return true;
+                    return false;
```

Comment:
Why did this change?

Reply:
whoops, just changed this so that the test would block and not end. Reverted.

Reply:
Ah, just changed this so the test would block and not exit. Reverted!